robot/description/urdf/constants.xacro

```diff
@@ -85,8 +85,7 @@
     <xacro:property name="laser_length" value="0.05"/>
     <xacro:property name="laser_width" value="0.05"/>
     <xacro:property name="laser_height" value="0.07"/>
-    <xacro:property name="laser_mass" value="0.16"/> 
-
```

Comment:
No need to remove this newline, it delimited that laser properties at a reasonable place